### PR TITLE
Added prizes and updated tracks

### PR DESCRIPTION
**Pull request by kjustin2**
Only issue is ensuring that we have the prizes set in stone for the 1st, 2nd, and 3rd place positions. Currently these prizes are the same as last year.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -203,8 +204,33 @@ <h2>Challenges</h2>
...
                             </section>
+							-->
+							
+							<section id="challenges" class="main special">
+                                <div class="spotlight">
+                                    <div class="content">
+                                        <header class="major">
+                                            <h2>Tracks</h2>
+                                        </header>
+										<h3><strong>Choose a track (or combine multiple!) to focus your idea upon.</strong></h3>
```
Comment (RitwikGupta) [inline index.html]: The indentation here and at line 218 onwards is really messed up. Fix this and then we can merge.

**Review by RitwikGupta — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (kjustin2) [reply index.html]: Well that's odd, something must have changed my indentation during upload.
I'll hop on fixing that up.

**Review by RitwikGupta — APPROVED** [review]

**Comment by RitwikGupta** [comment]
Reverted. Just realized this breaks the tracks section. No videos come up.

**Comment by kjustin2** [comment]
Hello,


The videos don't come up because we do not have videos for these tracks yet.


Those videos that are not being shown are from last year's event, and do not

reflect the changes made in this year's competition.


Thanks,


Justin Kramer

________________________________

**Comment by RitwikGupta** [comment]
Fair, reverted the revert 😁

**Pull request merged**